modules/rules_erlang/3.2.0/presubmit.yml

```diff
@@ -0,0 +1,24 @@
+matrix:
+  platform:
+  - ubuntu2004
+  - macos
+tasks:
+  verify_targets:
+    name: Verify build targets
+    platform: ${{ platform }}
+    environment:
+      ERLANG_VERSION: "24.3"
+    shell_commands:
+    - curl -O https://raw.githubusercontent.com/kerl/kerl/master/kerl
+    - chmod a+x kerl
+    - ./kerl update releases
+    - ./kerl build ${ERLANG_VERSION}
+    - ./kerl install ${ERLANG_VERSION} ~/kerl/${ERLANG_VERSION}
+    - |
+        ERLANG_HOME="$(realpath ~/kerl/${ERLANG_VERSION})"
+        cat << EOF >> user.bazelrc
+          build --//:erlang_version=${ERLANG_VERSION}
+          build --//:erlang_home=${ERLANG_HOME}
+        EOF
+    build_targets:
```

Comment:
This should be `'@rules_erlang//...'`
See 
https://github.com/bazelbuild/bazel-central-registry/blob/7bd963dc3755d14267c357134dadf39093730aca/modules/rules_erlang/2.5.2/presubmit.yml#L26

Also please fix this for 3.1.0: https://github.com/bazelbuild/bazel-central-registry/blob/main/modules/rules_erlang/3.1.0/presubmit.yml#L24

Reply:
Thanks for spotting this.